pontoon/base/models.py

```diff
@@ -2811,8 +2847,8 @@ def serialize(self):
             'approved': self.approved,
             'rejected': self.rejected,
             'fuzzy': self.fuzzy,
-            'error_count': self.errors.count(),
-            'warning_count': self.warnings.count(),
+            'errors': [error.message for error in self.errors.all()],
```

Comment:
[nit]  what do you think about`self.errors.values_list('message', flat=True)` here?

Reply:
That was my first idea, but it results in additional DB queries even if errors and warnings are prefetched, which is expensive.

For example, here:
https://github.com/mozilla/pontoon/blob/398e983e09d38bfd4dad2d943ab9bd5e6bddea58/pontoon/base/models.py#L2507